### DISCUSSION TITLE
Convert all colors in an image to a specific color - PHP

### DISCUSSION THREAD

**Discussion by etnic90** [ImageMagick/ImageMagick#2891] [Development]
Hi,
Using php I would like to make sure that all non-transparent pixels of a PNG image are converted to a color. For example I would like to convert all the pixels of an image to this color: # FFD700.

Thank you very much

**Comment by fmw42** [comment]
In command line, one method is:

`convert image.png +transparent "#FFD700" result.png`

**Reply by etnic90** [reply]
Thankyou. How can i translate your answer in php?

**Reply by fmw42** [reply]
Have you looked at the PHP Imagick documentation?

See https://www.php.net/manual/en/imagick.transparentpaintimage.php

**Reply by etnic90** [reply]
yes but I have some difficulties otherwise I would not have written here for help
